src/bundle/Resources/public/js/scripts/core/dropdown.js

```diff
@@ -294,6 +295,12 @@
                 item.classList.toggle('ibexa-dropdown__item--hidden', !isItemVisible);
             });
 
+            groups.forEach((group) => {
+                const isHidden = !group.querySelectorAll('.ibexa-dropdown__item:not(.ibexa-dropdown__item--hidden)').length;
```

Comment:
nitpick: no need to query all elements:
```suggestion
                const isAnyItemVisible = !!group.querySelector('.ibexa-dropdown__item:not(.ibexa-dropdown__item--hidden)');
```
also, `isHidden` is not a perfect name, because we are checking whether it _should be_ hidden. I think that `isAnyItemVisible` will reflect what we are actually checking in a better way.